response.go

```diff
@@ -300,7 +200,11 @@ func (r *Response) SenderAction(action string) error {
 		Recipient:    r.to,
 		SenderAction: action,
 	}
+	return r.PostToMessenger(&m)
+}
 
+// PostToMessenger posts the message to messenger, return the error if there's any
+func (r *Response) PostToMessenger(m interface{}) error {
```

Comment:
Let's change the  name of this function to `DispatchMessage` or maybe even just `Dispatch`